entity-framework/efcore-and-ef6/porting/index.md

```diff
@@ -1,61 +1,71 @@
 
-This is a non-exhaustive list of some changes in behavior between EF6 and EF Core. It is important to keep these in mind as your port your application as they may change the way your application behaves, but will not show up as compilation errors after swapping to EF Core.
+- Use new features in EF Core. There will be no new features added to EF6. All of the new functionality, for example the [Azure Cosmos DB provider](/ef/core/providers/cosmos/) and [`DbContextFactory`](/ef/core/what-is-new/ef-core-5.0/whatsnew#dbcontextfactory), will only be added EF Core.
 
-### DbSet.Add/Attach and graph behavior
+- Modernize your application stack by seamlessly integrating your data access with technologies like gRPC and GraphQL.
 
-In EF6, calling `DbSet.Add()` on an entity results in a recursive search for all entities referenced in its navigation properties. Any entities that are found, and are not already tracked by the context, are also marked as added. `DbSet.Attach()` behaves the same, except all entities are marked as unchanged.
+Regardless of your reasons for considering an upgrade, make sure that EF Core has all the features you need to use in your application. See [Feature Comparison](xref:efcore-and-ef6/index) for a detailed comparison of how the feature set in EF Core compares to EF6. If any required features are missing, ensure that you can compensate for the lack of these features before porting to EF Core.
 
-**EF Core performs a similar recursive search, but with some slightly different rules.**
+## Migrations
 
-* The root entity is always in the requested state (added for `DbSet.Add` and unchanged for `DbSet.Attach`).
-* **For entities that are found during the recursive search of navigation properties:**
-  * **If the primary key of the entity is store generated**
-    * If the primary key is not set to a value, the state is set to added. The primary key value is considered "not set" if it is assigned the CLR default value for the property type (for example, `0` for `int`, `null` for `string`, etc.).
-    * If the primary key is set to a value, the state is set to unchanged.
-  * If the primary key is not database generated, the entity is put in the same state as the root.
+This documentation uses the terms _port_ and _upgrade_ to avoid confusion with the term [_migrations_](/ef/core/managing-schemas/migrations/) as a feature of EF Core. Migrations in EF Core are fundamentally different than [EF6 Code First migrations](/ef/ef6/modeling/code-first/migrations/). There is not a recommend approach to port your migrations history, so plan to start "fresh" in EF Core. You can maintain the codebase and data from your EF6 migrations. Apply your final migration in EF6, then create an initial migration in EF Core. You will be able to track history in EF Core moving forward.
```

Comment:
Not sure if I would say they are "fundamentally different". More that the implementation has been significantly improved, even though they are fundamentally the same thing.

Reply:
This is good, but we should plan on providing more detailed instructions here. (Probably already done below, or planned.)